src/roadmap/v1/lifecycle/app_streams.py

```diff
@@ -91,6 +91,7 @@ def update_support_status(self):
             start_date=self.start_date,  # pyright: ignore [reportArgumentType]
             end_date=self.end_date,  # pyright: ignore [reportArgumentType]
             current_date=today,  # pyright: ignore [reportArgumentType]
+            months=6,
```

Comment:
I was thinking where this happens for rhel - I see it's default value added into `_calculate_support_status` [here](https://github.com/RedHatInsights/digital-roadmap-backend/commit/fc13ff7d0bed8bf47f61343c6ddd9b9406f7984a#diff-7082acc41212109bf5a24f7197b84d1500a833bd89832a3b2088cedc9f72a452R120)

Maybe I would prefer setting it each time manually to be more explicit where which value is used. But I'm definitely fine with this too -> approving those changes